PerimeterXModule/PxModule.cs

```diff
@@ -292,12 +293,20 @@ private void ResponseBlockPage(PxContext pxContext)
 			var config = (PxModuleConfigurationSection)ConfigurationManager.GetSection(PxConstants.CONFIG_SECTION);
 			string template = "block";
 			string content;
-			if (captchaEnabled)
+            if (pxContext.BlockAction == "j")
+            {
+                template = "challenge";
+            }
+            else if (captchaEnabled)
```

Comment:
new flow, captcha enabled should be deprecated
use BlockAction to decide what to render, j for challenge, c for captcha, else render block

Reply:
Fixing as part of the [funCaptcha PR](https://github.com/PerimeterX/perimeterx-asp-net/pull/22): https://github.com/PerimeterX/perimeterx-asp-net/commit/c1e080a0909eabe7fed95367055e0376f0f2695f